.github/workflows/publish.yml

```
@@ -35,5 +35,19 @@ jobs:
         env:
-          NODE_AUTH_TOKEN: ${{ secrets.GH_PAT }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
         run: npx semantic-release
+
+      - name: Create a new branch for release updates
+        run: |
+          git checkout -b release
+          git push origin release
+
+      - name: Create Pull Request from release to main
+        uses: peter-evans/create-pull-request@v3
```

actionlint (1.7.4)

the runner of "peter-evans/create-pull-request@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)